common/src/main/scala/org/neo4j/spark/util/Neo4jOptions.scala

```diff
@@ -295,7 +295,7 @@ case class Neo4jDriverOptions(
 
   def toDriverConfig: Config = {
     val builder = Config.builder()
-      .withUserAgent(s"neo4j-spark-connector/${Neo4jUtil.connectorVersion}")
+      .withUserAgent(s"neo4j-spark-connector/${Neo4jUtil.connectorVersion}/${Neo4jUtil.connectorEnv}")
```

Comment:
maybe avoid using `/` as a separator between connectionVersion and the databricks info, as this goes against the recommendation in https://neo4j.com/docs/bolt/current/bolt/message/#messages-hello
```suggestion
      .withUserAgent(s"neo4j-spark-connector/${Neo4jUtil.connectorVersion} (${Neo4jUtil.connectorEnv})")
```

Reply:
Done